src/common/CustomIconButton/CustomIconButton.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import PropTypes from 'prop-types';
+import {
+  EuiButton,
+} from '@elastic/eui';
+
+const CustomIconButton = ({
+  iconType, href, target, label,
```

Comment:
Let's keep the EuiButton contract. Label should be `children`.

Reply:
Done.